pytorch_lightning/trainer/callback_hook.py

```diff
@@ -36,7 +36,7 @@ class TrainerCallbackHookMixin(ABC):
     callbacks: List[Callback] = []
     lightning_module: "pl.LightningModule"
 
-    def on_before_accelerator_backend_setup(self) -> None:
+    def setup(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule", stage: Optional[str] = None) -> None:
```

Comment:
this should not be changed. this was the old convention for how callback hooks were called